build.gradle

```diff
@@ -51,6 +51,7 @@ ext {
 }
 
 def androidToolsVersion = '25.1.2'
+def butterknifeVersion = '8.4.0'
```

Comment:
This should not be in the root build file. It conflates the meaning of what's in `gradle.properties`